discover/index.html

```diff
@@ -30,9 +30,9 @@ <h5>Choose the Right Form</h5>
 <p>To obtain access to Discover, you will need to fill out one of the following forms <strong>(A or B)</strong>:</p>
 <ol type="A">
   <li>If you're requesting access on behalf of one of Utah’s: <strong>cities, counties, special districts, state agencies, K-12 or higher education institutions, or tribal entities</strong>, or if you are a <strong>contractor and formal partner</strong> of one of these organizations, you are covered by our license agreement to access imagery provided by Google on Discover. In this case, please read and complete the <a href="https://docs.google.com/a/utah.gov/forms/d/e/1FAIpQLScL5uUQIvw7op_ZcF4bijxcoOMGhNF0MXwJNGqSXS6IbjbKhA/viewform">Organizational Usage Agreement Form</a> to obtain access. You can read more about our Google Imagery license <a href="{{ "/discover/license/" | prepend: site.baseurl }}">here</a>.
-  <li>If you are <strong>not</strong> part of one of the organizations listed above, please complete the <a href="https://docs.google.com/a/utah.gov/forms/d/e/1FAIpQLScvASb37-R9WeFHNUsbIYEcVzQ_ceT__G4PZUaCx_xZxTuEpA/viewform">Discover Server Access</a> form to get access to all of the non-licensed base maps and imagery services.
+  <li>If you are <strong>not</strong> part of one of the organizations listed above, please complete the <a href="https://docs.google.com/a/utah.gov/forms/d/e/1FAIpQLScvASb37-R9WeFHNUsbIYEcVzQ_ceT__G4PZUaCx_xZxTuEpA/viewform">Discover Server Access Form</a> to get access to all of the nonlicensed base maps and imagery services.
 </ol>
-<p>Once you have completed either form (<strong>Please only fill out one</strong>.), you will receive information about the services and a link to access the Discover server. 
+<p>Once you have completed either form (<strong>please only fill out one</strong>), you will receive information about the services and a link to access the Discover Server. 
```

Comment:
So the article was a bit inconsistent with capping/lowercasing "Discover server"--should it be capped or lowercased? (Either is fine by me, as long as we're consistent. I think I tried to cap all of them, but if I should change them, let me know!)

Reply:
Same thing for "Google Service"--should that be capped? And again for "Utah imagery service"

Reply:
I think all of those items can be capitol.

Reply:
Got it! K, I'll make sure they're consistent.